### PR TITLE
Prevents body from scrolling when mobile menu is open

### PR DESCRIPTION
**Pull request by helenb**
@chris-lawton please could you review - I've used the body-scroll-lock plugin as per your suggestion.

### CODE REVIEW

File: src/components/header/header.js [hunk 1]
```diff
@@ -12,7 +13,31 @@ import { ReactComponent as Flame } from '@images/tbx-flame.svg'
 import styles from './header.module.scss'
 
 class Header extends React.Component {
-  state = { mobileNavOpen: false}
+
+  constructor(props) {
+    super(props)
+    this.state = { mobileNavOpen: false}
```
Comment (chris-lawton) [inline src/components/header/header.js]: picky, but a space before the closing curly would appease my ocd

File: src/components/header/header.js [hunk 2]
```diff
@@ -12,7 +13,31 @@ import { ReactComponent as Flame } from '@images/tbx-flame.svg'
...
+  componentDidMount() {
+    this.mobileMenuElement = this.mobileMenuRef.current
+  }
+
+  componentWillUnmount() {
+    clearAllBodyScrollLocks()
+  }
+
+  mobileMenuHandleClick() {
+    console.log(this.state.mobileNavOpen)
```
Comment (chris-lawton) [inline src/components/header/header.js]: presume this console.log can go now?

**Comment by chris-lawton** [comment]
Tested and working as expected 👍

File: src/components/header/header.js (hunk 1 above)
Comment (helenb) [reply src/components/header/header.js]: Happy to oblige :)

File: src/components/header/header.js (hunk 2 above)
Comment (helenb) [reply src/components/header/header.js]: whoops, ues

**Pull request merged**